test/DefaultCluster.Tests/BasicActivationTests.cs

```diff
@@ -186,12 +186,13 @@ public void BasicActivation_MultipleGrainInterfaces()
         public void BasicActivation_Reentrant_RecoveryAfterExpiredMessage()
         {
             List<Task> promises = new List<Task>();
-            TimeSpan prevTimeout = this.Client.ResponseTimeout;
+            var client = (IInternalClusterClient) this.Client;
+            TimeSpan prevTimeout = client.ResponseTimeout;
             try
             {
                 // set short response time and ask to do long operation, to trigger expired msgs in the silo queues.
                 TimeSpan shortTimeout = TimeSpan.FromMilliseconds(1000);
-                this.Client.ResponseTimeout = shortTimeout;
+                client.ResponseTimeout = shortTimeout;
```

Comment:
if this is the only reason why ResponseTimeout was kept in the interface, I suggest you just use the service provider from within this tests to get to the relevant component and tweak it there, since it's not something we want to encourage (even if these tests do a little bit of white-box testing)